pkg/controller/baremetalhost/host_state_machine.go

```diff
@@ -41,7 +41,7 @@ func (hsm *hostStateMachine) handlers() map[metal3v1alpha1.ProvisioningState]sta
 		metal3v1alpha1.StateInspecting:            hsm.handleInspecting,
 		metal3v1alpha1.StateExternallyProvisioned: hsm.handleExternallyProvisioned,
 		metal3v1alpha1.StateMatchProfile:          hsm.handleMatchProfile,
-		metal3v1alpha1.StateReady:                 hsm.handleReady,
+		metal3v1alpha1.StateAvailable:             hsm.handleAvailable,
```

Comment:
We can just add another line like:

    metal3v1alpha1.ProvisioningState("ready"): hsm.handleAvailable,

That won't change the current state, but will stop anything from breaking.

Reply:
Fixed